### PR TITLE
Readjust CAP constraints

### LINKED ISSUE #211: [PE-D][Tester D] Zero-CAP user accepted, contradicting error message
This error pops up

However, someone with a CAP of 0 (which is non-positive) gets accepted.

Try this command:
`add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 c/0.00/4.00 g/male u/Nanyang Polytechnic gd/05-2024 m/Computer Science ji/173296 jt/Software Engineer Intern t/rejected t/KIV`

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: cheeheng/ped#4

### LINKED ISSUE #225: [PE-D][Tester D] Big CAP allowed and registers as infinity
Command: `add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 c/1e310/1e309 g/male u/Nanyang Polytechnic gd/05-2024 m/Computer Science ji/173296 jt/Software Engineer Intern t/rejected t/KIV`

Perhaps it would be better to reject very high CAP (e.g. > 100) because it is most likely an error?

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: cheeheng/ped#9

### LINKED ISSUE #242: Fix v1.3 bugs for Jessica

### PR DESCRIPTION
**Pull request by maryjess**
Applying new constraints for CAP field:
- Rejecting values of 0 and below. Rationale: in reality, CAP values cannot be 0.
    - Resolves #211 
- Rejecting values of more than 100. Rationale: in reality, a CAP cannot exceed 100 and will prevent users from entering huge values for CAP.
    - Resolves #225 
   
Resolves #242

### CODE REVIEW

**Review by cliftonfelix — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/person/Cap.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public Cap(double cap, double max) {
      * @param max A maximum value for the CAP.
      */
     public static boolean isValidCap(double cap, double max) {
-        return cap <= max && cap >= MINIMUM;
+        return cap <= max && cap > MINIMUM && max <= 100.0;
```
Comment (lulucopter) [inline src/main/java/seedu/address/model/person/Cap.java:39]: Should we update this limit into the UG as well?

**Review by lulucopter — APPROVED** [review]
LGTM. Want to get your thoughts on updating the UG with the new limit as well

File: src/main/java/seedu/address/model/person/Cap.java (hunk 1 above)
Comment (maryjess) [reply src/main/java/seedu/address/model/person/Cap.java:39]: Will do, thanks for pointing that out!

Comment (maryjess) [reply src/main/java/seedu/address/model/person/Cap.java:39]: Pushed new commits to #258

**Pull request merged**